mindjunkies/lecture/views.py

```
@@ -141,11 +155,22 @@
         saved_lecture = form.save(commit=False)
         saved_lecture.course = self.course
         saved_lecture.module = self.module
-        saved_lecture.save()
-        messages.success(self.request, "Lecture created successfully!")
-        return redirect(
-            f"{reverse('lecture_home', kwargs={'course_slug': self.course.slug})}?module_id={self.module.id}"
-        )
+        # try to check if the lecture order is valid
+        try:
+            saved_lecture.full_clean()  
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)